datasets/seq_cifar10.py

```diff
@@ -61,152 +59,189 @@ def __getitem__(self, index: int) -> Tuple[Image.Image, int, Image.Image]:
 
         return img, target, not_aug_img
 
-class DriftingCIFAR10(CIFAR10):
+    def select_classes(self, classes_list: list[int]):
+        if len(classes_list) == 0:
+            self.data = np.array([])
+            self.targets = np.array([])
+            self.classes = []
+            return
+
+        mask = np.zeros_like(np.array(self.targets))
+        for label in classes_list:
+            mask = np.logical_or(mask, np.array(self.targets) == label)
+        self.data = self.data[mask]
+        self.targets = np.array(self.targets)[mask]
+
+        self.classes = classes_list
+
+    def apply_drift(self, classes: list):
+        if len(set(self.classes).union(classes)) == 0:
+            return
+        self.drifted_classes.extend(classes)
+
+        # TODO: figure out how to apply drift based on transform multiple times
+        # maybe we should change transforms or change the drift severity?
+
```

Comment:
This is something we should think about before moving to the experiments with new scenarios - how to apply drift in cifar10 and cifar100 multiple times.

Reply:
Multiple types of drifts or the same drift with multiple intensities, both seem like logical approaches we can try out.